packages/koa-metrics/src/test/Metrics.test.ts

```diff
@@ -27,6 +27,7 @@ describe('Metrics', () => {
   it('passes the host, port, prefix, and global tags to the statsd client', () => {
     const metrics = new Metrics(defaultOptions);
 
+    expect(metrics).toBeDefined();
```

Comment:
Why did this need to be added? I'm guessing just because it's not getting used directly? 👍 I'm fine with this if that's the case

Reply:
Yea ... ```'metrics' is declared but its value is never read.```